python-lib/language_dict.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+"""Module with constants defining the language support of underlying NLP libraries"""
+
+SUPPORTED_LANGUAGES_PYCLD3 = {
```

Comment:
No need for this one, it's used only for language detection

Reply:
done in 94885e4